active/website.md

```diff
@@ -0,0 +1,63 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.
+- Members can work on bugs
+- Members can tag,label and respond to issues.
+- Member can maintain and monitor the website including updating versions
+- Mentor new contributors to the website
+- Chair, Co-Chair and Board Liaison can sign off on new features
+
+## Initial membership
+
+- Chair: TBA
+- Co-Chair: TBA
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBA
+- Other members:
+    - Sarah Abderemane
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
+
+
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django and should be able to work with Django. Most importantly members must be willing to adhere to Django's [Code of Conduct](https://www.djangoproject.com/conduct/) . Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
```

Comment:
```suggestion
Members must have interest in Django and should be able to work with Django. Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
```

Following the code of conduct is a fundamental requirement of all Django community interactions so doesn’t need calling out.

Reply:
This website is pretty crucial to the Django community so I’d recommend further requirements. Possibly a requirement to be a DSF Individual Member.

Reply:
Won't this block new community members?